crates/bevy_dev_tools/src/ui_debug_overlay/inset.rs

```diff
@@ -184,9 +184,9 @@ impl<'w, 's> InsetGizmo<'w, 's> {
                 Vec2::new(right, bottom),
                 Vec2::new(right, top),
                 Vec2::new(left, top),
-            ];
-            self.draw
-                .linestrip_2d(strip.map(|v| self.relative(v)), color);
+            ]
+            .map(|v| self.relative(v));
+            self.draw.linestrip_2d(strip, color);
```

Comment:
This shows an instance where the use of `DerefMut` can cause compilation errors.
This issue of partial borrows is not exclusive `DerefMut` though, hopefully rust can get be smarter about this in the future with polonius.